app/store/chatList.go

```diff
@@ -97,6 +97,24 @@ func (s *Sessions) GetMessageList(id string) (error, *MessageList) {
 			Session: s.GetSession(index),
 		}
 		err := DS.Dbx.Select(&messageList.Messages, messagesSelectWhere, messageList.Session.ID)
+		if err != nil {
+			log.Errorln("[axolotl] get messagelist", err)
+			return err, nil
+		}
+		// attach the quoted messages
+		for i, m := range messageList.Messages {
```

Comment:
This could be extracted to a helper function because it's used at least twice

Reply:
Can you please do it? :)

Reply:
I think this is still true.